### PR TITLE
many firestore paths are used, hopefully this clarifies them a little

### PR DESCRIPTION
**Pull request by scytacki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,40 @@ Besides the parameters needed for loading data, the report also supports
 * `enableFirestorePersistence=true`: Uses a local firestore DB for data persistance across sessions and tabs. Clear the
                       DB by going to `dev tools > Application > IndexedDB > firebaseLocalStorageDb > Delete database`
 
+## FireStore Paths
+
+The portal-report loads and saves documents in report-service Firestore database.
```
Comment (kswenson) [inline README.md:140]:
```suggestion
The portal-report loads and saves documents in the report-service Firestore database.
```

File: README.md [hunk 2]
```diff
@@ -135,6 +135,40 @@ Besides the parameters needed for loading data, the report also supports
...
+LARA also writes data into the report-service.
+The Activity Player reads and writes data into the report-service.
+The new researcher report system reads data from the report-service.
+
+The following collections make up the report-service
```
Comment (kswenson) [inline README.md:145]:
```suggestion
The following collections make up the report-service:
```

File: README.md [hunk 3]
```diff
@@ -135,6 +135,40 @@ Besides the parameters needed for loading data, the report also supports
...
+
+`sources/${source}/resources`
+Each document represents an activity or sequence that can be reported on.
+Each document lists the reportable questions or items and what pages they are on.
+It also include information about what the correct answer is.
```
Comment (kswenson) [inline README.md:150]:
```suggestion
It also includes information about what the correct answer is.
```

File: README.md [hunk 4]
```diff
@@ -135,6 +135,40 @@ Besides the parameters needed for loading data, the report also supports
...
+`sources/${source}/user_settings/${validFsId(platformUserId)}/resource_link/${validFsId(resourceLinkId)}`
+Each document represents the settings for a teacher viewing the report. Currently
+these settings are which questions should be visible, and whether the user names should be anonymized.
+Unlike most other collections the `source` in this case is the platform. In most cases that is
+learn.concord.org
+
+`sources/${source}/feedback_settings`
+Each document has info about the feedback setting for a particular assignment (resourceLink).
+There can be a cached version of the rubric.
+And for each question there can be, scoreEnabled, feedbakcEnabled,
```
Comment (kswenson) [inline README.md:164]:
```suggestion
And for each question there can be `scoreEnabled`, `feedbackEnabled`
```

**Review by kswenson — APPROVED** [review]
👍 Looks good -- this kind of information is very helpful to have documented.

**Review by mklewandowski — APPROVED** [review]
👍 This is very useful!  Thanks for adding this!

**Pull request merged**